index.php

```diff
@@ -17,5 +21,9 @@
         <p>Associated to objects getting stuck in the cutlery drawer</p>
         <p>Carries no specific object</p>
     </div>
+
+    <!-- this is just to test the database connection functions -->
+    <?=print_r($deities);?>
```

Comment:
I think this syntax is to replace an echo statement? so it looks like you're doing an `echo print_r(...);` ?

Reply:
otherwise looks good to me, if we manage to figure out the issue with the database.